keyboards/planck/keymaps/priyadi/rules.mk

```diff
@@ -8,15 +8,24 @@ EXTRAKEY_ENABLE = yes       # Audio control and System control(+450)
 CONSOLE_ENABLE = no         # Console for debug(+400)
 COMMAND_ENABLE = no         # Commands for debug and configuration
 NKRO_ENABLE = yes            # Nkey Rollover - if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
-BACKLIGHT_ENABLE = yes      # Enable keyboard backlight functionality
 MIDI_ENABLE = no            # MIDI controls
 AUDIO_ENABLE = no           # Audio output on port C6
 UNICODE_ENABLE = no         # Unicode
 UNICODEMAP_ENABLE = yes     # Unicode map
 BLUETOOTH_ENABLE = no       # Enable Bluetooth with the Adafruit EZ-Key HID
 RGBLIGHT_ENABLE = no        # Enable WS2812 RGB underlight. 
 API_SYSEX_ENABLE = no
+
+ifeq ($(strip $(KEYBOARD)), planck/rev4)
 FAUXCLICKY_ENABLE = yes
+BACKLIGHT_ENABLE = yes
+endif
+
+ifeq ($(strip $(KEYBOARD)), planck/rev6)
+FAUXCLICKY_ENABLE = no
+BACKLIGHT_ENABLE = no
+DFU_ARGS = -d 0483:df11 -a 0 -s 0x08000000:leave
```

Comment:
I don't think that this is needed, as it's defined for the keyboard itself (or at least for the MCU)

Reply:
fixed in https://github.com/qmk/qmk_firmware/pull/5706/commits/be8b28f6fe7e47550d431edbcee521dde50bda72

Reply:
Well, specifically, I meant the DFU ARGS.  

It's defined here: https://github.com/qmk/qmk_firmware/blob/master/quantum/stm32/proton_c.mk#L44